src/engine/client/steam.cpp

```diff
@@ -34,7 +34,7 @@ class CSteam : public ISteam
 		if(pConnect[0])
 		{
 			NETADDR Connect;
-			if(net_addr_from_str(&Connect, pConnect) == 0)
+			if(net_host_lookup(pConnect, &Connect, NETTYPE_IPV4) == 0)
```

Comment:
It seems that this should allow for IPv6 lookups as well.
```suggestion
			if(net_host_lookup(pConnect, &Connect, NETTYPE_ALL) == 0)
```